app/lib/actions/katello/environment/library_create.rb

```diff
@@ -8,12 +8,7 @@ def plan(library_env)
                                                         :name => "Default Organization View",
                                                         :organization => library_env.organization)
 
-          ::Katello::ContentViewVersion.create! do |v|
-            v.content_view = library_view
-            v.major = 1
-          end
-
-          version = library_view.versions.first
+          version = library_view.versions.find_by(:major => 1)
```

Comment:
@jlsherrill @ehelms we weren't sure on this one. In tests, it looks like CVV version 1.0 of the DOV is already created, which makes this `create!` fail. We aren't sure if the CVVs on the DOV are actually used anywhere or not.

Reply:
I feel a bit dumb asking this, what does DOV stand for?

Reply:
default organization view, it is the content view that is just Library